packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-number/gux-form-field-number.tsx

```diff
@@ -188,6 +189,7 @@ export class GuxFormFieldNumber {
                 'gux-disabled': this.disabled,
                 'gux-clear': showClearButton
               }}
+              onClick={() => focusInputElement(this.input)}
```

Comment:
Is there a way to change the cursor to the `text` style also when you hover over the padding?